system.js

```diff
@@ -0,0 +1,117 @@
+const blank = document.getElementById("blank");
+const buyLottoButton = document.querySelector(".buyLottoButton");
+
+function generateNumber() {   //로또 생성 함수임
+  const lotto = [];
+  while(lotto.length < 6){
+    let num = Math.floor(Math.random() * 45) + 1;
+    if (!lotto.includes(num)) {
+      lotto.push(num);
+    }
+  }
+  return lotto.sort((a, b) => a - b);
+}
+
+function generateNumbers(lottoCount) {    //생성한 로또를 input만큼 배열에 담기기
+  let lottoes = [];
+  for (let i = 0;i<lottoCount;i++) {
+    lottoes.push(generateNumber());
+  }
+  return lottoes;
+}
+
+function matchNumber(arr1, arr2){
+  return arr1.filter(num => arr2.includes(num).length);
+}
+
+const main = document.querySelector(".mainbox");
+main.addEventListener("DOMContentLoaded", ()=>{
+  for(let i = 1;i<7;i++){
+    const cpuNumber = document.createElement("span");
+    cpuNumber.className = `ball cpuNum${i}`;
+    cpuNumber.textContent = `${cpuPick[i-1]}`;
+    console.log(cpuNumber);
+
+    cpuNumbers.appendChild(cpuNumber);
+  }
+})
+
+let myPick=[];
+
+buyLottoButton.addEventListener("click", function(){    //로또 구매 버튼 눌렀을 때 이벤트
+  let wantLottoCount = parseInt(blank.value);
+  const money = document.querySelector(".money");
+
+  if(wantLottoCount >= 1){    //버튼클릭시 번호 생성버튼 색깔 바뀌는거+총 금액 바뀌는거
+    createNumberButton.style.backgroundColor = '#4B89DC';
+    money.textContent = wantLottoCount + ",000";
```

Comment:
```suggestion
    money.textContent = (wantLottoCount * 1000).toLocaleString("ko-KR");
```
위 코드처럼 작성해보는 것은 어떨까요?🧐
지금 wantLottoCount가 문자열이 아닌 숫자인 것 같은데, 숫자 + 문자열로 작성하지 않고 이렇게 작성하면 더 직관적이고 좋을 것 같아요!